src/DevChatter.Bot.Core/Commands/SimpleCommand.cs

```diff
@@ -10,6 +11,9 @@ namespace DevChatter.Bot.Core.Commands
 {
     public class SimpleCommand : DataEntity, IBotCommand
     {
+        private DateTimeOffset _timeCommandLastInvoked;
+        public TimeSpan Cooldown => TimeSpan.Zero;
```

Comment:
Probably ok for now, but we'll likely want these in the database for simple commands.